llvm/test/Transforms/LoopPredication/pr66382.ll

```diff
@@ -1,12 +1,31 @@
-; XFAIL: *
```

Comment:
I don't think we usually land failing tests in tree (unless policies have changed? I'm not doing a ton of reviewing these days...). This makes it a bit harder to comment on them in the review.

Reply:
I've seen it done before. Regarding assertions - without them the crash would be just division by zero. Otherwise, the buildbots would complain about XFAIL passing.